### ISSUE TITLE
unit tests not passing

### ISSUE THREAD

**Issue by eroux** [BuddhistDigitalResourceCenter/lds-pdi#49]
`mvn test` doesn't pass, here are some highlights from the console:

```
Running io.bdrc.ldspdi.test.LdsTest
java.nio.file.NoSuchFileException: /home/marc/lds-queries/public/prefixes.txt
	at sun.nio.fs.UnixException.translateToIOException(UnixException.java:86)
	at sun.nio.fs.UnixException.rethrowAsIOException(UnixException.java:102)
	at sun.nio.fs.UnixException.rethrowAsIOException(UnixException.java:107)
	at sun.nio.fs.UnixFileSystemProvider.newByteChannel(UnixFileSystemProvider.java:214)
	at java.nio.file.Files.newByteChannel(Files.java:361)
	at java.nio.file.Files.newByteChannel(Files.java:407)
	at java.nio.file.Files.readAllBytes(Files.java:3152)
	at io.bdrc.ldspdi.service.ServiceConfig.initForTests(ServiceConfig.java:87)
```

```
[main] ERROR org.apache.jena.riot - loading remote context failed: http://purl.bdrc.io/context.jsonld
org.apache.jena.riot.RiotException: loading remote context failed: http://purl.bdrc.io/context.jsonld
	at org.apache.jena.riot.system.ErrorHandlerFactory$ErrorHandlerStd.error(ErrorHandlerFactory.java:140)
	at org.apache.jena.riot.lang.JsonLDReader.read$(JsonLDReader.java:162)
	at org.apache.jena.riot.lang.JsonLDReader.read(JsonLDReader.java:101)
	at org.apache.jena.riot.RDFParser.read(RDFParser.java:352)
	at org.apache.jena.riot.RDFParser.parseURI(RDFParser.java:321)
	at org.apache.jena.riot.RDFParser.parse(RDFParser.java:295)
	at org.apache.jena.riot.RDFParserBuilder.parse(RDFParserBuilder.java:500)
	at io.bdrc.ldspdi.test.LdsTest.getModelFromFileName(LdsTest.java:321)
	at io.bdrc.ldspdi.test.LdsTest.testJSONLDFormatter(LdsTest.java:211)
```

**Comment by MarcAgate** [comment]
using 
`mvn -Dtest="io.bdrc/ldspdi/test/**" test `

I get:
```
Results :
Tests run: 12, Failures: 0, Errors: 0, Skipped: 0 
```
I therefore cannot reproduce the issue. Re-pushing...

**Comment by MarcAgate** [comment]
Obviously, I couldn't reproduce the issue... since the failing path was my local path :grin: 
Fixed it.

**Comment by MarcAgate** [comment]
Fixed as of commit 25f7b13

**Comment by MarcAgate** [comment]
Resolved by commit ce9f7e6
